### ISSUE TITLE
QuickLook not added to path on MSI install

### ISSUE THREAD

**Issue by KRACK-BIT** [QL-Win/QuickLook#1283]
**To Reproduce**
1. Run `winget install QL-Win.QuickLook`
2. Launch Quicklook
3. Open terminal
4. Run `QuickLook.exe /path/to/image.jpg`

**Expected behavior**
`QuickLook.exe` should launch a preview of `/path/to/image.jpg`. Instead, no `QuickLook.exe` is found

As a temporary solution, I can add `%AppData%\Local\Programs\QuickLook` to my path; however, this is not done out-of-the-box

**Desktop:**
 - OS Version: [e.g. Windows 11 22H2]
 - QuickLook Version: [e.g. 3.7.3 from MSI, installed via WinGet]

**Screenshots and Exception Log**
```
PS C:\> QuickLook.exe
QuickLook.exe : The term 'QuickLook.exe' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ QuickLook.exe
+ ~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (QuickLook.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by xupefei** [comment]
I don't agree `QuickLook.exe` should be added to PATH. QuickLook is designed to be invoked by the spacebar rather than from the command line.

**Comment by pixkk** [comment]
Issue exist only in PowerShell?

**Comment by xupefei** [comment]
It's not necessary to call QuickLook.exe when it's running. Use this Bridge.exe (https://github.com/QL-Win/QuickLook/wiki/Develop,-build-and-integrate#integrating-with-file-managers) which will communicate with QuickLook via a named pipe.

Bridge.exe is not tight to a specific QL version. You can add it to PATH without polluting it.

**Comment by KRACK-BIT** [comment]
> It's not necessary to call QuickLook.exe when it's running. Use this Bridge.exe (https://github.com/QL-Win/QuickLook/wiki/Develop,-build-and-integrate#integrating-with-file-managers) which will communicate with QuickLook via a named pipe.
> 
> Bridge.exe is not tight to a specific QL version. You can add it to PATH without polluting it.

Thanks, that's good to know; I'll use that instead!
